modules/network-observability-release-notes-1-2-0-preparing-for-next-update.adoc

```diff
@@ -0,0 +1,11 @@
+// Module included in the following assemblies:
+// * network_observability/network-observability-release-notes-archive.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="network-observability-release-notes-1-2-0-preparing-for-next-update_{context}"]
+= Network observability release notes 1.2.0 preparing for the next update
+
+[role="_abstract"]
+Switch the Network Observability Operator's update channel from the deprecated `v1.0.x` to the `stable` channel to continue receiving future releases and updates.
+
```

Comment:
I think this is too long of short description. See the guidance: https://docs.google.com/document/d/1xRc_licMTbLrLG9vmJ4bP9aPk6mzwwly5lGlkRz9d9s/edit?tab=t.0#heading=h.35i82kfdktwb